core/linux-tools/build

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh -uxe
+
+export KBUILD_BUILD_HOST=`hostname`
+export KBUILD_BUILD_USER=`id -nu`
+export CC=gcc
+export CXX=g++
```

Comment:
`CC` and `CXX` are already defined [here](https://github.com/mdartmann/mkissconfig/blob/main/etc/profile.d/flags.sh#L6)